File: test/Data.Analyzers.Vsix.Test/VisualBasic/MissingImplementationAttribute.vb
Imports DevZest.Data.Annotations

<CheckConstraint(DuplicateModelAttribute._CK_AlwaysTrue, "CK")>
Public Class MissingImplementationAttribute
    Inherits Model

    Friend Const _CK_AlwaysTrue = NameOf(CK_AlwaysTrue)
    Private ReadOnly Property CK_AlwaysTrue As _Boolean
        Get
            Return _Boolean.Const(True)
        End Get
    End Property
End Class

File: test/Data.Analyzers.Vsix.Test/CSharp/MissingImplementationAttribute.cs
using DevZest.Data.Annotations;

namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    [CheckConstraint(nameof(CK_AlwaysTrue), "CK")]
    public class MissingImplementationAttribute : Model
    {
        private _Boolean CK_AlwaysTrue
        {
            get { return _Boolean.Const(true); }
        }
    }
}
